bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/targetComponent/TargetFeedbackEditor.scala

```diff
@@ -21,13 +21,14 @@ class TargetFeedbackEditor extends TelescopePosEditor {
   def getComponent: Component = tab.peer
 
   override def edit(ctxOpt: GOption[ObsContext], target: SPTarget, node: ISPNode): Unit = {
-    val analysis = {
-      val mt = OT.getMagnitudeTable
-      ctxOpt.asScalaOpt.map(TargetGuidingFeedback.targetAnalysis(_, mt, target)).getOrElse(Nil)
-    }
-
     // Construct the rows for the table. Optionally a BAGS row, and then a list of AGS analysis rows.
     val rows = {
+      val analysisRows = for {
+        c <- ctxOpt.asScalaOpt.toList
+        a <- TargetGuidingFeedback.targetAnalysis(c, OT.getMagnitudeTable, target)
+        if !target.isTooTarget
```

Comment:
Aha... I think I got it, now that you pointed me to those implicits for `Boolean`. I believe that `fold` should do the trick!

`val analysisRows = target.isTooTarget fold (Nil, ctxOpt.asScalaOpt.toList.flatMap(TargetGuidingFeedback.targetAnalysis(_, OT.getMagnitudeTable, target)))`